### PR TITLE
add configurable GA property

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: config-production.js-dist [hunk 1]
```diff
@@ -10,6 +10,7 @@ var config = {
     host: 'localhost',
     theme: 'basic-theme',
     domain: 'http://contribs.jsunconf.eu',
-    eventname: 'JS Unconf'
+    eventname: 'JS Unconf',
+    analyticscode: ''
```
Comment (robin-drexler) [inline config-production.js-dist]: How about naming it more expressive something like `googleAnalyticsUaCode` maybe?

Reading it, I thought one should enter the entire analytincs js snippet here.

**Comment by robin-drexler** [comment]
Just noticed that the tests are failing. I already tried restarting the build. Would you like to have a look?

**Comment by robertkowalski** [comment]
all green now!

**Comment by robin-drexler** [comment]
\o/ lgtm

**Comment by robertkowalski** [comment]
merged as 3b99e04921b3fa9cd28a3618bbbcb40fd9277e05

**Pull request closed without merging**